packages/cli/src/types/config.ts

```diff
@@ -0,0 +1,78 @@
+import type { Sink } from "@apibara/indexer";
+import type {
+  C12InputConfig,
+  ConfigWatcher,
+  ResolvedConfig,
+  WatchConfigOptions,
+} from "c12";
+import type { WatchOptions } from "chokidar";
+import type { NestedHooks } from "hookable";
+import type { DeepPartial } from "./_utils";
+import type { ApibaraHooks } from "./hooks";
+import type { RollupConfig } from "./rollup";
+
+/**
+ * Apibara Config type (apibara.config)
+ */
+export interface ApibaraConfig<
+  // biome-ignore lint/complexity/noBannedTypes: <explanation>
+  T extends Record<string, Partial<ApibaraConfig<T>>> = {},
+> extends DeepPartial<Omit<ApibaraOptions<T>, "preset" | "presets">>,
+    C12InputConfig<ApibaraConfig> {
+  sink?: {
+    default: Sink;
+    [key: string]: Sink;
+  };
+  dev?: boolean;
```

Comment:
What does `dev` do?

Reply:
> What does dev do?

oh my bad, it should be just in `ApibaraOptions` and not in `ApibaraConfig`. 
its more like an internal flag, to determine if its a dev command or build command.